src/groupelements.jl

```diff
@@ -136,8 +147,18 @@ findindex(::SectorValues{ZNElement{N, p}}, c::ZNElement{N, p}) where {N, p} = c.
 Base.hash(c::ZNElement, h::UInt) = hash(c.n, h)
 Base.isless(c1::ZNElement{N, p}, c2::ZNElement{N, p}) where {N, p} = isless(c1.n, c2.n)
 
-# TODO: is this true?
-# const AbelianGroupElement{G, ω} = AbstractGroupElement{G, ω} where {G <: AbelianGroup}
-# BraidingStyle(::Type{<:AbelianGroupElement}) = HasBraiding()
-
-BraidingStyle(::Type{<:AbstractGroupElement}) = NoBraiding()
+# Experimental
+BraidingStyle(::Type{ZNElement{2, 0}}) = Bosonic()
+BraidingStyle(::Type{ZNElement{2, 1}}) = Fermionic()
+BraidingStyle(::Type{ZNElement{N, 0}}) where {N} = Bosonic()
+BraidingStyle(::Type{ZNElement{N, p}}) where {N, p} = Anyonic()
+function Rsymbol(a::ZNElement{N, p}, b::ZNElement{N, p}, c::ZNElement{N, p}) where {N, p}
+    if p == 0
+        R = 1
+    elseif N == 2 && p == 1
+        R = ifelse(a.n == b.n == 1, -1.0, 1.0)
```

Comment:
In principle we could specialize the cocycle and Rsymbol for this case to return $\pm1$ as `Int` if I'm not mistaken, but this might not really be worth it

Reply:
Yes, I had this at first. But it seems that also for SU2Irrep we return ±1.0 so I opted for floating point. Of course, in that case the Fsymbol definitely needs floating point numbers, so yes, the `N ==2 && p == 1` case could use integers for both `Fsymbol` (`cocycle`) and `Rsymbol`.